keyboards/mysticworks/wyvern/wyvern.h

```diff
@@ -0,0 +1,129 @@
+) \
+{ \
+    { k00, k01, k02, k03, k04, k05, k06, k07, k08, k09 }, \
+    { k10, k11, k12, k13, k14, k15, k16, k17, XXX, k19 }, \
+    { k20, k21, k22, k23, k24, k25, k26, k27, k28, k29 }, \
+    { k30, k31, k32, k33, k34, k35, k36, k37, XXX, k39 }, \
+    { XXX, k41, k42, k43, k44, k45, k46, k47, XXX, k49 }, \
+    { k50, k51, k52, k53, k54, k55, k56, k57, XXX, k59 }, \
+    { k60, k61, k62, k63, k64, k65, k66, k67, k68, k69 }, \
+    { k70, k71, XXX, k73, k74, k75, k76, k77, XXX, k79 }, \
+    { XXX, XXX, k82, k83, XXX, k85, XXX, k87, XXX, k89 }, \
+    { k90, k91, k92, XXX, XXX, XXX, XXX, k97, k98, k99 }  \
+}
+
+#define LAYOUT_iso_numpad_right( \
+    k00, k10, k01, k11,   k02, k12, k03, k13, k04, k14, k05, k15, k06, k16, k07, k17, k08, k09, k19, \
+    k20, k30, k21, k31,   k22, k32, k23, k33, k24, k34, k25, k35, k26, k36, k27, k37, k28,      k39, \
+    k40, k50, k41,        k42, k52, k43, k53, k44, k54, k45, k55, k46, k56, k47, k57, k58, k49, k59, \
+    k60, k70, k61, k71,   k62, k72, k63, k73, k64, k74, k65, k75, k66, k76, k67, k77, k68, k69, k79, \
+    k80,      k81,        k82, k92, k83,                k85,                k87, k97, k98, k89, k99  \
+) \
+{ \
+    { k00, k01, k02, k03, k04, k05, k06, k07, k08, k09 }, \
+    { k10, k11, k12, k13, k14, k15, k16, k17, XXX, k19 }, \
+    { k20, k21, k22, k23, k24, k25, k26, k27, k28, XXX }, \
+    { k30, k31, k32, k33, k34, k35, k36, k37, XXX, k39 }, \
+    { k40, k41, k42, k43, k44, k45, k46, k47, XXX, k49 }, \
+    { k50, XXX, k52, k53, k54, k55, k56, k57, k58, k59 }, \
+    { k60, k61, k62, k63, k64, k65, k66, k67, k68, k69 }, \
+    { k70, k71, k72, k73, k74, k75, k76, k77, XXX, k79 }, \
+    { k80, k81, k82, k83, XXX, k85, XXX, k87, XXX, k89 }, \
+    { XXX, XXX, k92, XXX, XXX, XXX, XXX, k97, k98, k99 }  \
+}
+
+#define LAYOUT_iso_numpad_left( \
+    k00, k10, k01, k11,   k02, k12, k03, k13, k04, k14, k05, k15, k06, k16, k07, k17, k08, k09, k19, \
+    k20, k30, k21, k31,   k22, k32, k23, k33, k24, k34, k25, k35, k26, k36, k27, k37, k28,      k39, \
+         k50, k41, k51,   k42, k52, k43, k53, k44, k54, k45, k55, k46, k56, k47, k57, k58, k49, k59, \
+    k60, k70, k61, k71,   k62, k72, k63, k73, k64, k74, k65, k75, k66, k76, k67, k77, k68, k69, k79, \
+         k90,      k91,   k82, k92, k83,                k85,                k87, k97, k98, k89, k99  \
+) \
+{ \
+    { k00, k01, k02, k03, k04, k05, k06, k07, k08, k09 }, \
+    { k10, k11, k12, k13, k14, k15, k16, k17, XXX, k19 }, \
+    { k20, k21, k22, k23, k24, k25, k26, k27, k28, XXX }, \
+    { k30, k31, k32, k33, k34, k35, k36, k37, XXX, k39 }, \
+    { XXX, k41, k42, k43, k44, k45, k46, k47, XXX, k49 }, \
+    { k50, k51, k52, k53, k54, k55, k56, k57, k58, k59 }, \
+    { k60, k61, k62, k63, k64, k65, k66, k67, k68, k69 }, \
+    { k70, k71, k72, k73, k74, k75, k76, k77, XXX, k79 }, \
+    { XXX, XXX, k82, k83, XXX, k85, XXX, k87, XXX, k89 }, \
+    { k90, k91, k92, XXX, XXX, XXX, XXX, k97, k98, k99 }  \
+}
```

Comment:
There needs to be a newline at the end of this file.
```
keyboards/mysticworks/wyvern/wyvern.h:111:33: error: backslash-newline at end of file [-Werror]
```

Reply:
Fixed